app/src/main/java/com/weezlabs/smsresponder/utils/SmsListener.java

```diff
@@ -0,0 +1,67 @@
+package com.weezlabs.smsresponder.utils;
+
+
+import android.annotation.SuppressLint;
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.provider.Telephony;
+import android.support.v4.app.NotificationCompat;
+import android.telephony.SmsMessage;
+
+import com.weezlabs.smsresponder.MainActivity;
+import com.weezlabs.smsresponder.R;
+import com.weezlabs.smsresponder.utils.MessageUtil;
+import com.weezlabs.smsresponder.utils.SmsIntentService;
+
+public class SmsListener extends BroadcastReceiver {
+
+    public static final String MESSAGE_KEY = "com.weezlabs.smsresponder.action.SMS_MESSAGE";
+    public static final int NOTIFY_ID = 12345;
+
+    @SuppressLint("NewApi")
```

Comment:
You set minApi=15, but you use feature with minApi=19, and don't care about lower versions. You should never do that. Here you can just set minApi=19 for whole project and remove @SuppressLint("NewApi")